lib/glimesh_web/templates/layout/_navbar.html.leex

```diff
@@ -31,7 +31,11 @@
           <li class="nav-item dropdown user-profile-dropdown">
               <a href="javascript:void(0);" class="nav-link dropdown-toggle user" id="userProfileDropdown"
                  data-toggle="dropdown" aria-haspopup="true" aria-expanded="true">
-                  <%= @current_user.username %>
+                  <%= if @current_user.displayname do %>
```

Comment:
Do we need this if statement if the user always has a displayname?

Reply:
I can't remember for the life of me why I did that but definitely not needed.